media-sound/denemo/denemo-2.2.0.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools xdg-utils
+
+DESCRIPTION="A music notation editor"
+HOMEPAGE="http://www.denemo.org/"
+SRC_URI="mirror://gnu/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3+ OFL-1.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# configure options currently not used:
+# --enable-mem(no) memory debugging: needs Electric fence (efence), which
+#		is not available in portage. See https://github.com/boundarydevices/efence
+# --enable-gtk-doc-pdf(no) doesn't work
+IUSE="alsa +aubio debug evince doc jack +fluidsynth gtk3 nls +portaudio
```

Comment:
BTW any reason not to enable GTK+3 by default?

Reply:
No, I think I chose to not enable it by default, so user's choice from possible make.conf settings get used. I can change it.

Reply:
I think for end-user packages we want to default to whichever GUI is better supported upstream. If both work the same, we want to prefer the newer version not to force user to install old packages by default.

Reply:
Ok, that sounds reasonalbe :+1: